dvc/repo/brancher.py

```diff
@@ -24,21 +29,22 @@ def brancher(  # noqa: E302
 
     scm = self.scm
 
-    if self.scm.is_dirty():
-        from dvc.scm.tree import WorkingTree
-
-        self.tree = WorkingTree(self.root_dir)
+    is_dirty = scm.is_dirty()
+    self.tree = WorkingTree(self.root_dir)
+    if is_dirty:
         yield "Working Tree"
+    else:
+        yield CLEAN_WORKING_TREE_SIGNATURE.format(scm.active_branch())
```

Comment:
Why not just `working tree`? What if you are on a tag not a branch?

Reply:
or even in detached-head state.

Reply:
just having `working tree` for all of those cases seems reasonable to me

Reply:
I think it might be confiusing to user that it displays only `working tree` , and it might be good to include current branch if it is not dirty

Reply:
on the other side, branch will not change too much